ParkingLotApi/Dtos/ParkingLotDto.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading.Tasks;
+using ParkingLotApi.Entities;
+using System.ComponentModel.DataAnnotations;
+
+namespace ParkingLotApi.Dtos
+{
+    public class ParkingLotDto
+    {
+        public ParkingLotDto()
+        {
+        }
+
+        public ParkingLotDto(ParkingLotEntity parkingLotEntity)
+        {
+            Name = parkingLotEntity.Name;
+            Capacity = parkingLotEntity.Capacity;
+            Location = parkingLotEntity.Location;
+            Cars = parkingLotEntity.Cars.Select(carEntity => new CarDto(carEntity)).ToList();
```

Comment:
as the coach clarified, cars may not be included in the parkinglot data, because the order include numberplate; the order data can be separated with parkinglot